Extension/src/LanguageServer/references.ts

```diff
@@ -325,6 +338,8 @@ export class ProgressHandler {
         this.referencesViewFindPending = false;
         this.referencesChannel.clear();

```

Comment:
Put these in a function that clears all views?

        this.referencesChannel.clear();
        this.findAllRefsView.show(false);
        this.renameView.show(false);